collect_app/src/test/java/org/odk/collect/android/utilities/RegexMatcher.java

```diff
@@ -0,0 +1,34 @@
+package org.odk.collect.android.utilities;
+
+import org.hamcrest.Description;
+import org.hamcrest.TypeSafeMatcher;
+
+/**
+ * Created by simran on 4/10/2017.
+ */
+
+public class RegexMatcher extends TypeSafeMatcher<String> {
+
+    private final String regex;
+
+    public RegexMatcher(final String regex) {
```

Comment:
I like “final” as much as—or more than—the next guy, but in the parameter list, when not needed, they don’t have good utility/noise ratios, in my opinion.